src/main/java/taplink/network/menu/api/model/City.java

```diff
@@ -0,0 +1,20 @@
+package taplink.network.menu.api.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "CITY")
+public class City extends BaseEntity {
+
+    @Column(name = "CODE", nullable = false, unique = true)
+    private String code;
+
+    @Column(name = "NAME", nullable = false, unique = true)
+    private String name;
+
+    @Column(name = "SORT")
+    private Integer sort;
```

Comment:
What does this column mean? Can we change it to isSorted and the type is boolean?

Reply:
And the same for other same fields sort.